molgenis-data-i18n/src/test/java/org/molgenis/data/i18n/LanguageRepositoryDecoratorTest.java

```diff
@@ -43,7 +43,7 @@ public void testAddExistingLanguage()
 		verify(delegateRepository).add(language);
 	}
 
-	@Test(expectedExceptions = AddLanguageException.class)
+	@Test(expectedExceptions = RepositoryCapabilityException.class, expectedExceptionsMessageRegExp = "repository:Language capability:WRITABLE")
```

Comment:
Filter the WRITABLE capability from the capabilities listed by getCapabilities()

Reply:
I don't understand what you mean with this ...

Reply:
I meant that if the repository is going to throw an exception that says you cannot write to it, it should not claim that you can write to it when asked about its capabilities. But as discussed, the repository is "somewhat writable" :)